tools/tools-core/src/test/java/org/wso2/carbon/tools/dropins/DropinsDeployerToolTest.java

```diff
@@ -215,41 +214,38 @@ private static void createOtherProfiles() throws IOException {
     }
 
     private static void prepareCarbonHomeForDropinsTests() throws IOException {
-        String dropinsFolder = Constants.DROPINS;
-        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, dropinsFolder,
+        String libFolder = Constants.LIB;
+        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, libFolder,
                 TestConstants.ARTIFACT_ONE),
-                Paths.get(carbonHome.toString(), Constants.OSGI_REPOSITORY, dropinsFolder, TestConstants.ARTIFACT_ONE));
-        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, dropinsFolder,
+                Paths.get(carbonHome.toString(), libFolder, TestConstants.ARTIFACT_ONE));
+        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, libFolder,
                 TestConstants.ARTIFACT_TWO),
-                Paths.get(carbonHome.toString(), Constants.OSGI_REPOSITORY, dropinsFolder, TestConstants.ARTIFACT_TWO));
-        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, dropinsFolder,
+                Paths.get(carbonHome.toString(), libFolder, TestConstants.ARTIFACT_TWO));
+        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, libFolder,
                 TestConstants.ARTIFACT_THREE),
-                Paths.get(carbonHome.toString(), Constants.OSGI_REPOSITORY, dropinsFolder,
-                        TestConstants.ARTIFACT_THREE));
-        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, dropinsFolder,
+                Paths.get(carbonHome.toString(), libFolder, TestConstants.ARTIFACT_THREE));
+        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, libFolder,
                 TestConstants.ARTIFACT_FOUR),
-                Paths.get(carbonHome.toString(), Constants.OSGI_REPOSITORY, dropinsFolder,
-                        TestConstants.ARTIFACT_FOUR));
-        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, dropinsFolder,
+                Paths.get(carbonHome.toString(), libFolder, TestConstants.ARTIFACT_FOUR));
+        Files.copy(Paths.get(TestConstants.TARGET_FOLDER, TestConstants.TEST_RESOURCES, libFolder,
                 TestConstants.ARTIFACT_FIVE),
-                Paths.get(carbonHome.toString(), Constants.OSGI_REPOSITORY, dropinsFolder,
-                        TestConstants.ARTIFACT_FIVE));
+                Paths.get(carbonHome.toString(), libFolder, TestConstants.ARTIFACT_FIVE));
     }
 
     private static List<BundleInfo> getExpectedBundleInfo() {
-        String dropinsFolder = Constants.DROPINS;
+        String libFolder = Constants.LIB;
```

Comment:
What would be the reason for assigning a constant to a local variable without using the constant itself?